### ISSUE TITLE
Tab order not saved

### ISSUE THREAD

**Issue by Moilleadoir** [sienori/Tab-Session-Manager#870]
## Short description
Every time the browser is restarted the tabs appear in a different order.

## Steps to reproduce
1. Exit browser
2. Start browser

## Expected result
Tab order should be the same as the previous session.

## Actual result
Tab order is randomly different. Sometimes tab are missing. Sometimes *pinned* tabs are missing.

## Platform information
* Platform (OS): Linux (Manjaro)
* Version of browser: 92.0
* Version of Tab Session Manager:  6.10.1

## Additional context
This has been happening for some months. Sometimes hunting through the saved sessions with find one with the correct ordering.

**Comment by sienori** [comment]
This may be caused by other extensions. Please try disabling other extensions.
